balancer/balancer.go

```diff
@@ -111,6 +111,8 @@ type NewSubConnOptions struct {
 	// CredsBundle is the credentials bundle that will be used in the created
 	// SubConn. If it's nil, the original creds from grpc DialOptions will be
 	// used.
+	// Deprecated: Use the Attributes field in resolver.Address to pass
```

Comment:
Please add a "//" line above this.  I'm not sure whether it's required, but it looks nicer anyway.

Reply:
Done.